adhocracy4/maps/mixins.py

```diff
@@ -0,0 +1,49 @@
+from django.conf import settings
+from easy_thumbnails.files import get_thumbnailer
+
+
+class MapItemListMixin(object):
+
+    def dump_geojson(self):
+        result = {}
+        result['type'] = 'FeatureCollection'
+        feature_list = []
+
+        for item in super().get_queryset():
+
+            url = ''
+
+            if hasattr(item, 'image') and item.image:
+                image = get_thumbnailer(item.image)['map_thumbnail']
+                url = image.url
+
+            properties = {
+                'name': item.name,
+                'slug': item.slug,
+                'image':  url,
+                'comments_count': item.comment_count,
+                'positive_rating_count': item.positive_rating_count,
+                'negative_rating_count': item.negative_rating_count,
+                'url': item.get_absolute_url()
+            }
+            point_dict = item.point
+            point_dict['properties'] = properties
+            feature_list.append(point_dict)
+
+        result['features'] = feature_list
+        return result
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context['mapideas_json'] = self.dump_geojson()
```

Comment:
It seems not to be json. But on the other hand it is directly outputted to the template in our [idea listing](https://github.com/liqd/a4-opin/blob/master/euth/maps/templates/euth_maps/mapidea_list.html#L7). Also it is marked safe and I think it is [not safe](https://opin-dev.liqd.net/en/projects/asdasdasd-3/). Security issues coming up!